### PR TITLE
KEYCLOAK-18075 - Remove "role_list" from expected default client scopes

### PR DESCRIPTION
**Pull request by Aboullos**

### CODE REVIEW

File: testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/clients/ClientClientScopesTest.java [hunk 1]
```diff
@@ -98,7 +98,7 @@ public void testSetupClientScopes() {
 
         // Retrieve client through adminClient
         found = findClientByClientId(TEST_CLIENT_ID);
-        Assert.assertNames(found.getDefaultClientScopes(), "email", "role_list"); // SAML client scope 'role_list' is included too in the rep
+        Assert.assertNames(found.getDefaultClientScopes(), "email"); // SAML client scope 'role_list' is included too in the rep
```
Comment (hmlnarik) [inline testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/clients/ClientClientScopesTest.java:101]: The comment is irrelevant now, can you please remove it?

Comment (Aboullos) [reply testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/clients/ClientClientScopesTest.java]: Done

**Review by hmlnarik — APPROVED** [review]
Thank you for your contribution!

**Comment by hmlnarik** [comment]
The failed tests are unrelated.

**Pull request merged**